hadoop-hdds/docs/content/security/SecurityWithRanger.md

```diff
@@ -46,6 +46,12 @@ Property|Value
 ozone.acl.enabled         | true
 ozone.acl.authorizer.class| org.apache.ranger.authorization.ozone.authorizer.RangerOzoneAuthorizer
 
+To use the RangerOzoneAuthorizer, you also need to add the following environment variables to ozone-env.sh:
+```
+export OZONE_CLASSPATH="${OZONE_HOME}/share/ozone/lib/*:${OZONE_HOME}/share/ozone/lib/libext/*:${OZONE_HOME}/share/ozone/lib/libext/ranger-ozone-plugin-impl/*"
+```
+* The location of the ranger-ozone-plugin jars depends on where the Ranger Plugin is installed.
```

Comment:
Would it be better to just state that the Ranger Ozone plugin (and its dependencies?) need to be added to `OZONE_CLASSPATH`, without the example with a specific path?

Reply:
Just to configure `export OZONE_CLASSPATH="${OZONE_HOME}/share/ozone/lib/libext/*"`, dependencies are automatically introduced.

Reply:
Sorry if my comment was confusing.  What I tried to say is that the example is too specific given that Ranger plugin can be installed anywhere.

I would find the following a bit more clear:

> Ranger Ozone plugin also needs to be added to Ozone's classpath via the `OZONE_CLASSPATH` variable.  Assuming the plugin is installed under `${OZONE_HOME}/share/ozone/lib/libext`,  add the following setting in `ozone-env.sh`:
>
> ```
> export OZONE_CLASSPATH="${OZONE_HOME}/share/ozone/lib/libext/*"
> ```

Reply:
This is true, so you need to copy Ranger Ozone plugin jars to the Ozone installation node.